utils/sort-by.js

```diff
@@ -1,13 +1,28 @@
+/*
+sort-by defaults to ascending order
+@param prop {String} The property to get from each item
```

Comment:
I was thinking that since this is a js file, it should match the JSdoc syntax
http://usejsdoc.org/tags-param.html

With everything else, @arg will be better anyway